packages/local-ai/src/main/kotlin/elide/runtime/localai/InferenceResults.kt

```diff
@@ -97,8 +97,8 @@ public sealed interface InferenceResults {
     /** @return Streamed inference results. */
     @JvmStatic
     public inline fun suspending(crossinline producer: suspend FlowCollector<String>.() -> Unit): Suspending {
-      return flow {
-        producer()
+      return flow<String> {
+        // producer()
```

Comment:
Is this commenting intentional?

Reply:
Yes, for now, the async layer is not using this code path. It's deadlocking/stack overflowing. So, the background thread runs through `inferSync` (still offloaded, though, from guest execution), and we can fix this later. It's a performance wart.